openapi-spec/swagger.yaml

```diff
@@ -200,6 +186,44 @@ paths:
         required: true
         schema:
           type: string
+      - name: limit
+        in: query
+        description: Number of query results on multiple pages
```

Comment:
you mean, this is the maximum results per page ?

Reply:
Yes. It's a query parameter.